sensu/transport/rabbitmq/rabbitmq.go

```diff
@@ -95,6 +157,25 @@ func (t *RabbitMQTransport) Connect() error {
 		return err
 	}
 
+	if prefetchString := config.Prefetch.String(); prefetchString != "" {
```

Comment:
Actually, that won't work properly, because `.Int64()` returns an `invalid syntax` error when `config.Prefetch == ""` and I wanted to avoid logging a warning in this case since that's a legit value if we don't specify the prefetch value in the config.